web/webkit/src/main/scala/net/liftweb/http/Templates.scala

```diff
@@ -28,7 +28,10 @@ import java.io.InputStream
  * Contains functions for obtaining templates
  */
 object Templates {
-  private val suffixes = LiftRules.templateSuffixes
+  // Making this lazy to ensure it doesn't accidentally init before Boot completes in case someone touches this class.
+  private lazy val parsers = LiftRules.contentParsers.flatMap(parser =>
```

Comment:
I guess it depends on how we suggest folks add to the list. Right now it's a `Seq`, which means there's not necessarily a prescribed way to add to it. If it were a `List`, the expectation would be that you'd use `::=` to add to it, thus meaning “first one wins” would actually make more sense. I can get behind that approach.

While I'm looking at it again, it's probably worth mentioning in the Scaladocs for `contentParsers` that it's read once when the first template is rendered and then ignored.